README.md

```diff
@@ -118,6 +118,7 @@ The following environment variables are supported in the default configuration:
 |APM_SERVERURL      | URL to the APM intake service. |
 |APM_SECRETTOKEN    | Secret token, if required. |
 |APM_USEROUTEURI    | `true` or `false` defaults to `true`. The default behavior is to record the URL as defined in your routes configuration. Set to `false` to record the requested URL, but keep in mind that this can result in excessive unique entries in APM. |
+|APM_IGNORE_PATTERNS| Ignore specific routes or jobs by transaction name. Should be a regular expression, and will match multiple patterns via pipe `\|` in the regex. Note that 4 backslashes should be used to match a single backslash. Example: `"/\/health-check\|^OPTIONS \|Foo\\\\Bar\\\\Job/"` |
```

Comment:
btw. considering that this pattern will likely have slashes in it, it might also be a nice option to use a different pattern delimiter. E.g. you could do `@/foo/bar@` rather than `/\/foo\/bar/`. No change required, since this is possible with the current setup.